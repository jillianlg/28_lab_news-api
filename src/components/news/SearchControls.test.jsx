import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Search from './SearchControls';


describe('Search component', () => {
  afterEach(() => cleanup());
  it('renders Search', () => {
    const { asFragment } = render(
      <Search 
        // id="searchInput"
        // type="text" 
        // placeholder="search" 
        onChange={jest.fn()}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});