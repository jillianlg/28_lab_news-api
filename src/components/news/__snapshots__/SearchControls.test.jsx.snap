// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders Search 1`] = `
<DocumentFragment>
  <label
    for="searchInput"
  >
    Search
  </label>
  <input
    id="searchInput"
    placeholder="search"
    type="text"
  />
</DocumentFragment>
`;
